src/services/car/__tests__/messageLead.test.ts

```diff
@@ -364,6 +367,80 @@ describe("Car API", () => {
     })
   })
 
+  describe("#fetchDealerWhatsappLeads", () => {
+    const { content, pagination } = PaginatedLeads([
+      SearchWhatsappLeadFactory(),
+    ])
+
+    beforeEach(() => {
+      fetchMock.mockClear()
+      fetchMock.mockResponse(
+        JSON.stringify({
+          content: content,
+          ...pagination,
+        })
+      )
+    })
+
+    it("calls correct endpoint", async () => {
+      await fetchDealerWhatsappLeads({
+        dealerId: 1234,
+        query: {
+          page: 2,
+          size: 7,
+          sort: defaultLeadSort,
+        },
+        options: {
+          accessToken: "DummyTokenString",
+        },
+      })
+
+      expect(fetch).toHaveBeenCalledWith(
+        expect.stringMatching(
+          /\/dealers\/1234\/whats-app-tracking-entries\?page=1&size=7&sort=createdDate%2Cdesc$/
+        ),
+        expect.any(Object)
+      )
+    })
+
+    it("should trow error if accessToken is not passed", async () => {
+      let error
+      try {
+        await fetchDealerCallLeads({
+          dealerId: 1234,
+          query: {
+            page: 0,
+            size: 7,
+            sort: {},
+          },
+        })
+      } catch (err) {
+        error = err
+      }
+
+      expect(fetch).not.toHaveBeenCalled()
+      expect(error).toBeDefined()
+      expect(error.message).toBeDefined()
```

Comment:
What's wrong with: 
```typescript
        expect(fetchDealerWhatsappLeads({
          dealerId: 1234,
          query: {
            page: 0,
            size: 7,
            sort: {},
          },
        })).rejects.toEqual("<error message>")
        expect(fetch).not.toHaveBeenCalled()
```
?

Reply:
Expected: "[Error: You tried to make an authenticated requests without providing an access token! Please pass a valid token as a request option.]"
    Received: [Error: You tried to make an authenticated requests without providing an access token!
     Please pass a valid token as a request option.]

      414 |           },
      415 |         })
    > 416 |       ).rejects.toEqual(
          |                 ^
      417 |         "[Error: You tried to make an authenticated requests without providing an access token! Please pass a valid token as a request option.]"
      418 |       )
      419 |       expect(fetch).not.toHaveBeenCalled()


it works like this 
```
  await expect(fetchDealerWhatsappLeads({
          dealerId: 1234,
          query: {
            page: 0,
            size: 7,
            sort: {},
          },
        })).rejects.toBeDefined()
        expect(fetch).not.toHaveBeenCalled()
```